config/paths.js

```diff
@@ -0,0 +1,13 @@
+const path = require( 'path' );
+
+const ASSETS = 'assets';
+
+const assetsManifestOutputPath = path.resolve( ASSETS, 'dist/build-manifest.json' );
+const assetsPath = `./${ ASSETS }/ZZZ/`;
```

Comment:
Added const here for consistency, but I am not  :100: happy with how readability ended :)

Reply:
oh ha ha ya we don't want to keep that as all of the new EDTR replacement files will still be in `/assets/src/`
the `ZZZ` folder was just a way to keep all of the existing logic untouched and out of the way while we refactor everything